echidna/core/fill_spectrum.py

```diff
@@ -1,3 +1,4 @@
+import root_help
```

Comment:
Also I would prefer not to have relative imports as they make it less clear what you are importing and where it's coming from see [more detailed arguments]( http://programmers.stackexchange.com/questions/159503/whats-wrong-with-relative-imports-in-python). But it's up to you if you want to change it or not.